pkg/manage/default-initial-data.json

```diff
@@ -19,6 +20,7 @@
     },
     "user": {
         "1": {
+            "id": 1,
```

Comment:
I think error-prone. I think It would be a good idea to auto create the id field. There is no case, where the key should not be set or should be another value.

Reply:
What about the following syntax:
```
{
  "user": {
    "270e4396-d609-4a30-a11f-dc8708ea37be": {
      "id": 1,
      "username": "admin"
    }
  }
}
```

Reply:
Another option is to use the current Osnabrück-format as is (with list including).

Reply:
I would still like map[string][int][string]interface{} and autocreate the id field. But the Osnabrück-format is also ok.

My problem with the Osnabrück-format is, that it is ok for this case but not ok in other situations. I would like to have one format for any each case.